RecoHI/HiEgammaAlgos/interface/EcalClusterIsoCalculator.h

```diff
@@ -0,0 +1,37 @@
+#ifndef HiEgammaAlgos_EcalClusterIsoCalculator_h
+#define HiEgammaAlgos_EcalClusterIsoCalculator_h
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "DataFormats/EgammaReco/interface/BasicClusterFwd.h"
+#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
+
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
+
+
+class EcalClusterIsoCalculator
+{
+public:
+
+  EcalClusterIsoCalculator(const edm::Event &iEvent, const edm::EventSetup &iSetup, const edm::Handle<reco::BasicClusterCollection> barrel, const edm::Handle<reco::BasicClusterCollection> endcap);
+
+  /// Return the ecal cluster energy in a cone around the SC
```

Comment:
Please let me know if the comment syntax for Doxygen is different. Do the comments go above or below the prototypes?